docs/guides/getting-started.md

```diff
@@ -0,0 +1,61 @@
+# Getting started
+
+_Note: the [example app](https://github.com/mapbox/mapbox-navigation-ios/blob/master/Examples/Swift/ViewController.swift) is also a great resource for viewing how to get started. It also has a few more advanced examples._
+
+**Before getting started, make sure you have installed the Navigation SDK, if not [follow these instructions](./installation).**
```

Comment:
If I’m not mistaken, the link has to point to `installation.md`, not just `installation`.